### ISSUE TITLE
Attribute Error when training for sentence pair task using Albert model

### ISSUE THREAD

**Issue by abhitaliyan** [ThilinaRajapakse/simpletransformers#146]
I am getting AttributeError when training for sentence pair task using Albert model. Dataset contains 3 columns: text_a, text_b, labels. Labels are 0 and 1. I am pasting the traceback screenshot.

![image](https://user-images.githubusercontent.com/11687870/72396309-21c81380-36f1-11ea-8d0c-538f15cf8ef2.png)

**Comment by ThilinaRajapakse** [comment]
Can you give me a minimal script to reproduce this? I'm not getting the error with a dummy dataset.

**Comment by abhitaliyan** [comment]
Below is the minimal script to reproduce the error. 

`
from simpletransformers.classification import ClassificationModel
import pandas as pd
import sklearn

train_data = [['Example sentence belonging to class 1' * 50, 'Example eval sentence belonging to class 1',1], 
              ['Example sentence belonging to class 0', 'Example eval sentence belonging to class 1',0], 
              ['Example  2 sentence belonging to class 0', 'Example eval sentence belonging to class 1',0]] + [['Example sentence belonging to class 0', 'Example eval sentence belonging to class 1',0] for i in range(12)]
train_df = pd.DataFrame(train_data, columns=['text_a', 'text_b','labels'])

eval_data = [['Example eval sentence belonging to class 1','Example eval sentence belonging to class 1', 1],
             ['Example eval sentence belonging to class 0', 'Example eval sentence belonging to class 1',0]]
eval_df = pd.DataFrame(eval_data, columns=['text_a', 'text_b','labels'])

train_args={
    'fp16': False,
    'sliding_window': False,
    'n_gpu': 2
}

# Create a TransformerModel
model = ClassificationModel('albert', 'albert-base-v2', args=train_args)

# Train the model
model.train_model(train_df, eval_df=eval_df)

# Evaluate the model
result, model_outputs, wrong_predictions = model.eval_model(eval_df, acc=sklearn.metrics.accuracy_score)
print(result)
`

**Comment by abhitaliyan** [comment]
I am getting similar error for simple classification tasks as well. Error comes when attempting training with Albert model. Script runs fine for roberta.

**Comment by ThilinaRajapakse** [comment]
I am not getting an error when running the script you provided either.

```
    There is an imbalance between your GPUs. You may want to exclude GPU 1 which
    has less than 75% of the memory or cores of GPU 0. You can do so by setting
    the device_ids argument to DataParallel, or by setting the CUDA_VISIBLE_DEVICES
    environment variable.
  warnings.warn(imbalance_warn.format(device_ids[min_pos], device_ids[max_pos]))
Epoch:   0%|                                                                                                                                                                                                                                                                                                              | 0/1 [00:00<?, ?it/s/
home/thilina/miniconda3/envs/skil/lib/python3.7/site-packages/torch/nn/parallel/_functions.py:61: UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.                                                                                     | 0/2 [00:00<?, ?it/s]
  warnings.warn('Was asked to gather along dimension 0, but all '
Current iteration: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:02<00:00,  1.25s/it]
Epoch: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:02<00:00,  2.51s/it]
Training of albert model complete. Saved to outputs/.
Converting to features started. Cache is not used.
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00, 45.90it/s]
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00, 57.91it/s]
/home/thilina/miniconda3/envs/skil/lib/python3.7/site-packages/sklearn/metrics/_classification.py:896: RuntimeWarning: invalid value encountered in double_scalars
  mcc = cov_ytyp / np.sqrt(cov_ytyt * cov_ypyp)
{'mcc': 0.0, 'tp': 0, 'tn': 1, 'fp': 0, 'fn': 1, 'acc': 0.5, 'eval_loss': 0.8698716163635254}
```

>UserWarning: Was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector.         

I assume this is because it's dummy data. It's still a warning for me and not an error.
